content/liveobjects/lifecycle.textile

```diff
@@ -0,0 +1,66 @@
+---
+title: Lifecycle Events
```

Comment:
Can this be sentence case please? In the table of contents too (and the same for 'Batch Operations'